db/migrate/20240109113342_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.references :user, foreign_key: { to_table: :users }
+      t.references :post, foreign_key: { to_table: :posts }
```

Comment:
- [ ] [Optional] There is no need to add `foreign_key: { to_table: :users }` and `:post, foreign_key: { to_table: :posts }` because Active Record will add the foreign keys as `user_id` and `post_id` automatically.
```suggestion
      t.references :user
      t.references :post
```

Reply:
Thank you very much for the review and the suggestion, it's highly noted.